tests/requirements.txt

```diff
@@ -0,0 +1,27 @@
+cookiecutter==0.9.0
+coverage==3.7.1
+ddt==0.8.1
+diff-cover==0.7.2
+Django==1.4.16
+django_nose==1.2
+fs==0.5.0
+lazy==1.2
+mock==1.0.1
+nose==1.3.4
+nose-parameterized==0.3.5
+pep8==1.5.7
+pylint==0.28
+pypng==0.0.17
+rednose==0.4.1
+requests==2.4.3
+selenium==2.44.0
+simplejson==3.6.5
+unicodecsv==0.9.4
+webob==1.4
```

Comment:
@mtyaka I understand the intent, but if these packages were to be pinned, they should be pinned in the package that requires them - ie in xblock-sdk. You can open a PR there to propose it, but I would also like to stay away from trying to handle the requirements of another repo from the XBlocks that depend on it. Otherwise it would mean duplicating these requirements in all the XBlock repos (and maintaining that list). Also, while it's true that it can make it harder to pin down the origin of a test failure when the upgrade isn't deliberate, it's not necessarily a waste of time - the test would still need to be fixed, eventually, and fixing it earlier ensures that we don't write any more tests that depend on the older version.